roles/common/tasks/monitoring.yml

```diff
@@ -241,6 +241,12 @@
   notify: restart sensu-client missing ok
   when: common.hwraid.enabled|bool
 
+- name: check IPMI sensors
+  sensu_check: name=check-ipmi-sensors plugin=check-ipmi-sensors.py
+               args="--criticality {{ monitoring.ipmi_checks.criticality }}" use_sudo=True
+  when: monitoring.ipmi_checks is defined and monitoring.ipmi_checks.enabled|default(False)|bool
```

Comment:
we should set `monitoring.ipmi_checks.enabled: false` in monitoring-common defaults so that we don't have to check that `monitoring.ipmi_checks` is defined here.